src/main/java/com/powsybl/openloadflow/network/impl/LfDanglingLineBranch.java

```diff
@@ -74,21 +78,26 @@ public List<BranchResult> createBranchResult(double preContingencyBranchP1, doub
     }
 
     @Override
-    public List<LfLimit> getLimits1(final LimitType type) {
+    public List<LfLimit> getLimits1(final LimitType type, LimitReductionManager limitReductionManager) {
         var danglingLine = getDanglingLine();
         switch (type) {
             case ACTIVE_POWER:
-                return getLimits1(type, danglingLine.getActivePowerLimits().orElse(null));
+                return getLimits1(type, danglingLine.getActivePowerLimits().orElse(null), limitReductionManager);
             case APPARENT_POWER:
-                return getLimits1(type, danglingLine.getApparentPowerLimits().orElse(null));
+                return getLimits1(type, danglingLine.getApparentPowerLimits().orElse(null), limitReductionManager);
             case CURRENT:
-                return getLimits1(type, danglingLine.getCurrentLimits().orElse(null));
+                return getLimits1(type, danglingLine.getCurrentLimits().orElse(null), limitReductionManager);
             case VOLTAGE:
             default:
                 throw new UnsupportedOperationException(String.format("Getting %s limits is not supported.", type.name()));
         }
     }
 
+    @Override
+    public List<Double> getLimitReductions(TwoSides side, LimitReductionManager limitReductionManager, LoadingLimits limits) {
+        return Collections.emptyList();
```

Comment:
Yes :-) My PR was just as minimal as possible because the plan is to use powsybl-core utility methods and not my work. Indeed this PR is just a MVP to validate the interface (which we did together and the utility methods) and to give something quickly to users but not definitive at all.